### ISSUE TITLE
Unexpected issue of interest with RNG

### ISSUE THREAD

**Issue by dorothealint** [ebeshero/DHClass-Hub#421]
So in playing around with this exercise, I apparently tried to choose a tag name that it didn't like!  There is no prior designation in the file or the schema that would make this name invalid that I can see.  When I changed it, the code goes green.  So I did "fix" the issue for submitting my assignment but I didn't figure out what it's issue was with it but I'm betting it's one of those things that I trip into that I shouldn't have even thought of for weeks that makes life so darn complicated.  Thought I'd put it here for interest sake and maybe some insights that I'm missing.

https://github.com/ebeshero/DHClass-Hub/blob/master/TroubleShooting/lint_rnc_ex2_trouble_file.rnc
https://github.com/ebeshero/DHClass-Hub/blob/master/TroubleShooting/lint_xml_for_rnc_ex2_trouble_file.xml

 @ebeshero @jonhoranic @alexfell06 @MarkTheShark01 @dorothealint @Pomilui @garrettjoiner @quantum-satire @setriplette

**Comment by ebeshero** [comment]
@dorothealint I just left a line comment on your file! I think this is just missing a pattern definition, but let me know if my suggestion doesn't solve the problem...

**Comment by dorothealint** [comment]
@ebeshero Oh! That makes complete sense!  I'm so glad it was something simple that I just couldn't see.

**Comment by ebeshero** [comment]
@dorothealint There *are* some words that Relax-NG won't accept as labels, but there are workarounds for that called "escapes"--I'll show you in class tomorrow!

**Comment by dorothealint** [comment]
@ebeshero Yep that worked!  I remembered to do the pattern definition for "name" after I changed it but totally missed that I didn't do one for slave in the original file.  When I changed it back to post it here and it still gave me the error, I even left the definition for "name" in there!
